### PR TITLE
Update NEWS and UPGRADING for next release

### PR DESCRIPTION
**Pull request by clepple**
Current as of master @ b8df06b761, assuming libusb-1.0+0.1 gets merged

@aquette or @jimklimov can you summarize the user-visible changes for daisychain and/or any of the other `snmp-ups` changes I might have missed?

@zykh let me know if I missed anything, but I think most of your changes were internal to the build.

### CODE REVIEW

**Comment by jimklimov** [comment]
There was a burst of development in the last sprint, now formally still in validation but practically passed, to add support for new firmware of Eaton G3 ePDUs. Some new fields were added in mib2nut mappings, which I gather are subject to an RFC process to agree that these names/patterns will stay used by the common upstream project. I wonder if 2.7.5 release should wait for this naming ritual, as well as backporting of that change (refer to https://github.com/42ity/nut/commits/FTY if interested in the gory details), to complete and so get a place in the news?

Also there were a few build/recipe related fixes, as well as some smaller PRs opened already, that I'd like to include before rolling a release if we do them so rarely ;)

**Comment by clepple** [comment]
@jimklimov I'll be honest, keeping up with the pace of changes in FTY has been quite tiring, and I really just want to see the libusb-1.0 stuff get released first before we address other large changes. Will existing NUT users be negatively impacted by not merging these changes now, considering that owners of newer hardware may prefer to use the FTY fork to get the latest validated changes anyway?

I think we can add a few smaller PRs in as well, but at some point we need to prioritize what we will release, and what will be deferred - and we need to be mindful that integration testing takes time (we can't become complacent just because the CI tools indicate everything built).

I would prefer not to wait this long for another release; hence my efforts to get the ball rolling again.

**Comment by aquette** [comment]
hi @clepple 
the impact for the user of not merging the FTY things for now is not a big deal. The main general-purpose improvements already landed into our mainline.
I'm also in favor of releasing 2.7.5 with the minimal set, postponing for example #415 to 2.7.6.
I'll take a look at the milestone and issues states today, in an effort to have 2.7.5 as soon as possible.

**Comment by aquette** [comment]
@clepple @zykh @jimklimov FYI I've resync the NEWS with the last commits
@jimklimov please complete the line on nut-driver-enumerator (and possibly systemd), thx:
https://github.com/networkupstools/nut/pull/486/commits/85b5a511297df84c934bfa703a3fcc3ec69a9839#diff-ff4e2dc4962dc25a1512353299992c8dR81

**Comment by aquette** [comment]
Note: the failure is on spellcheck due to the "@jim" entry.
@jimklimov please complete the line on nut-driver-enumerator (and possibly systemd), thx:
https://github.com/networkupstools/nut/pull/486/files#diff-ff4e2dc4962dc25a1512353299992c8dR81

**Comment by nico202** [comment]
Is there something blocking this? I'd really like to see a new release!
